### ISSUE TITLE
Accessibility Audit: Static site Misc sections

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5768]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the pages in the following miscellaneous sections of the static site. Do the page scan as well as any relevant guided tests.
Pages to visit:
- https://www.simplereport.gov/
- https://www.simplereport.gov/about-us/
- https://www.simplereport.gov/covid-19-results/
- https://www.simplereport.gov/contact-us
- https://www.simplereport.gov/support/
- https://www.simplereport.gov/exit-disclaimer/
- https://www.simplereport.gov/terms-of-service/
- https://www.simplereport.gov/unsupported-browser/
- https://www.simplereport.gov/waitlist/ 

Components to test:
- Header
- Footer
- Touchpoints modal

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by emyl3** [comment]
added this as well:
Components to test:
- Header
- Footer
- Touchpoints modal

**Comment by emyl3** [comment]
Opened tickets:
https://github.com/CDCgov/prime-simplereport/issues/5788
https://github.com/CDCgov/prime-simplereport/issues/5789
https://github.com/CDCgov/prime-simplereport/issues/5790
